apps/exploits/api.py

```diff
@@ -344,6 +347,46 @@ def get(self, request):
         )
 
 
+class ExploitsReportData(APIView):
+    """Export only the data required to generate the exploits report
+    TODO: Paginate
+    TODO: Check query efficiency
+    """
+
+    @extend_schema(
+        responses={
+            200: {
+                "type": "object",
+                "properties": {
+                    "exploits": {"type": "array", "items": {"type": "object"}},
+                    "flaws": {"type": "array", "items": {"type": "object"}},
+                    "trackers": {"type": "array", "items": {"type": "object"}},
+                    "affects": {"type": "array", "items": {"type": "object"}},
+                },
+            }
+        }
+    )
+    def get(self, request):
+        exploits = (
+            Exploit.objects.filter(source="CISA")
+            .filter(flaw__isnull=False)
+            .order_by("cve")
+        )
+
+        flaws = Flaw.objects.filter(exploit__in=exploits).distinct()
+        affects = Affect.objects.filter(flaw__in=flaws).distinct()
+        trackers = Tracker.objects.filter(affects_in=affects).distinct()
```

Comment:
There is only a single underscore so it is basically an unknown attribute **affects_in**. Plus I have no idea what does **in** operator applied on many-to-many relation so please double-check that this really works as you expect.